.github/workflows/unittests.yaml

```diff
@@ -10,11 +10,16 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-rc.1"]
+        exclude:
+          - os: macos-latest
+            python-version: "3.11.0-rc.1"
```

Comment:
Strangely, all of the `macos-latest` builds succeeded except for one running this release candidate version of Py3.11.

For that version, schema data extraction [failed](https://github.com/hhursev/recipe-scrapers/actions/runs/3221154009/jobs/5268750127#step:5:26) for the generic `wild_mode` scraper test.

Reply:
I'm having trouble figuring this one out.  Despite enabling debug logging, there wasn't much additional context in the `wild_mode` scraper test failure output.

Reply:
Roughly speaking: it's as if the unit test doesn't read the HTML content correctly (or at all?) when the [`TestWildMode`](https://github.com/hhursev/recipe-scrapers/blob/93c0c86a4a27697d34efcc1d04d9531af2df6223/tests/test_wild_mode.py) test runs.

The errors arise when the scraper-under-test attempts to retrieve parsed schema.org JSON-LD fields from the content.  For example, we [see](https://github.com/hhursev/recipe-scrapers/actions/runs/3221492074/jobs/5269493368#step:5:5790):

```
  File "/Users/runner/work/recipe-scrapers/recipe-scrapers/recipe_scrapers/_factory.py", line 21, in cook_time
    return self.schema.cook_time()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/recipe-scrapers/recipe-scrapers/recipe_scrapers/_schemaorg.py", line 120, in cook_time
    raise SchemaOrgException("Cooktime information not found in SchemaOrg")
recipe_scrapers._exceptions.SchemaOrgException: recipe-scrapers exception: Cooktime information not found in SchemaOrg
```

Note that this is happening in Py3.10 as well as seen in the [latest builds](https://github.com/hhursev/recipe-scrapers/actions/runs/3221492074/jobs/5269493368#step:5:22).